client/src/router.ts

```diff
@@ -75,14 +75,17 @@ export const router = new Router({
 router.beforeEach(async (to, _from, next) => {
     coreStore.setLoading(true);
     if (!coreStore.initialized) {
-        const response = await fetch("/api/auth");
-        if (response.ok) {
-            const data = await response.json();
-            if (data.auth) {
+        const authResponse = await fetch("/api/auth");
+        const versionResponse = await fetch("/api/version");
+        if (authResponse.ok && versionResponse.ok) {
```

Comment:
Now you block the second request by waiting on the first.  What you want to do is something akin to

```
const [authResponse, versionResponse] = await Promise.all([fetch("/api/auth"), fetch("/api/version"]);
coreStore.setVersion(versionResponse.version);
if (authResponse.auth) { ... }
```

Reply:
Latest commit has these changes.

I've actually never done this before, so it's new!